virttest/utils_libvirt/libvirt_nwfilter.py

```
@@ -0,0 +1,30 @@
+"""
+Accommodate libvirt nwfilter utility functions.
+
+:copyright: 2020 Red Hat Inc.
+"""
+
+import logging
+
+from virttest import virsh
+
+
+def clean_up_nwfilter_binding(ignore_status=True):
```

I believe it is better to have default option as `ignore_status=False` and can be overridden by user with known precautions.

---

API by default should not silently ignore the failure..

---

updated accordingly

---

Tnx!